projects/packages/connection/legacy/class-jetpack-xmlrpc-server.php

```diff
@@ -763,8 +762,8 @@ public function sync_object( $args ) {
 	 */
 	public function validate_urls_for_idc_mitigation() {
 		return array(
-			'home'    => Functions::home_url(),
-			'siteurl' => Functions::site_url(),
+			'home'    => Urls::home_url(),
```

Comment:
I think we need to add `use Automattic\Jetpack\Connection\Urls;` to this file.

Reply:
good catch no namespace on this class :)